.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
-target/*
\ No newline at end of file
+target/*
+hibernate.iml
```

Comment:
you may not write this, because you have `*.iml`

Reply:
I did it coz this file was added to the git repository even though *.iml